packages/react-native/Libraries/Components/Switch/Switch.d.ts

```diff
@@ -51,6 +51,11 @@ export interface SwitchProps extends SwitchPropsIOS {
    */
   thumbColor?: ColorValue | undefined;
 
+  /**
+   * Color of the active foreground switch grip.
+   */
+  activeThumbColor?: ColorValue | undefined;
```

Comment:
Hey @VickyA371 
Could you point me to the place where this prop is implemented? This is just the typescript type for it.

I believe this prop is not existing and this PR should be reverted

Reply:
Hi @cortinico 

I agree that this should be reverted. I did a thorough investigation in the android native code and found out that there is no such prop as activeThumbColor.

The reason I made this change was because I was working on a monorepo project and the prop activeThumbColor was working fine for react-native-web. However, when I added the import for react-native, I encountered some typescript errors that indicated that this prop was not supported.

Hence, I only modified the ts file to fix the errors.